src/app/collective-rewards/rewards/hooks/useGetBackerRewardPercentage.ts

```diff
@@ -23,7 +23,14 @@ export const useGetBackerRewardPercentage = (builder: Address, timestampInSecond
 
     const [previous, next, cooldownEndTime] = data
 
-    const percentageData = getPercentageData(previous, next, cooldownEndTime, timestampInSeconds)
+    const currentTimestamp = timestampInSeconds ?? Math.floor(Date.now() / 1000)
+    const current = currentTimestamp < cooldownEndTime ? previous : next
```

Comment:
Why did we remove this function if we have the same code here and in the `useGetBackersRewardPercentage`?